meinberlin/apps/budgeting/serializers.py

```diff
@@ -85,23 +85,27 @@ def get_session_token_voted(self, proposal):
         been voted with the token in the current session
         """
         if "request" in self.context:
-            if "voting_token" in self.context["request"].session:
+            if "voting_tokens" in self.context["request"].session:
+                module = self.context["view"].module
+                module_key = str(module.id)
                 token = None
-                try:
-                    token = VotingToken.objects.get(
-                        token=self.context["request"].session["voting_token"],
-                        module=self.context["view"].module,
-                    )
-                except VotingToken.DoesNotExist:
-                    pass
+                if module_key in self.context["request"].session["voting_tokens"]:
+                    try:
+                        token = VotingToken.objects.get(
+                            token=self.context["request"].session["voting_tokens"][
+                                module_key
+                            ],
+                            module=module,
+                        )
+                    except VotingToken.DoesNotExist:
+                        pass
 
                 vote = TokenVote.objects.filter(
                     token=token,
                     content_type=ContentType.objects.get_for_model(proposal.__class__),
                     object_pk=proposal.pk,
                 )
```

Comment:
This block can be indented one level more, right? If the module is not in the voting_tokens, then we will also not find a vote anyway.

Reply:
Ah yes, true!